website/e2e-test/smoke-test.spec.js

```diff
@@ -32,3 +32,35 @@ const assertUrlLoads = async (page, url) => {
 test('Renders homepage', async ({ page }) => {
   await assertUrlLoads(page, EMBLEM_URL);
 });
+
+test('Renders login', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/login');
+});
+
+test('Renders logout', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/logout');
+});
+
+test('Renders donate', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/donate?campaign_id=59d32e9805fc4d3388db');
```

Comment:
Should we move the `campaign_id` value to a separate variable?

e.g:
```
  const CAMPAIGN_ID = '59d32e9805fc4d3388db';
  await assertUrlLoads(page, EMBLEM_URL + `/donate?campaign_id=${CAMPAIGN_ID}`);
```

Reply:
Done